azext_iot/central/common.py

```diff
@@ -11,8 +11,13 @@
 
 from enum import Enum
 
+from azext_iot.central.models.enum import ApiVersion
+
 EDGE_ONLY_FILTER = "type eq 'GatewayDevice' or type eq 'EdgeDevice'"
 
+API_VERSION = ApiVersion.ga.value
+API_VERSION_PREVIEW = ApiVersion.preview.value
```

Comment:
wait so we are still using 2 api versions? Does the customer know which commands use which (though I am sure they wont care)

Reply:
Yeah, some APIs only exist in preview first and then integrate into GA version. So if they're using sth already in GA, then call GA otherwise preview. They're all in doc. And preview and GA here are both latest one.